modules/cms/src/main/CmsApi.scala

```diff
@@ -38,11 +38,7 @@ final class CmsApi(coll: Coll, markup: CmsMarkup)(using Executor):
 
   def update(prev: CmsPage, data: CmsForm.CmsPageData)(using me: Me): Fu[CmsPage] =
     val page = data update me
-    val idChange: Funit = (prev.id != page.id).so:
-      coll.exists($id(page.id)) flatMap:
-        case true  => fufail(s"A page with the ID ${page.id} already exists")
-        case false => coll.delete.one($id(prev.id)).void
-    idChange >> coll.update.one($id(page.id), page, upsert = true) inject page
+    coll.update.one($id(page.id), page) inject page
```

Comment:
@ornicar You have a conflict before https://github.com/lichess-org/lila/commit/aea0cd7b76be36e944374b003fe2b390463d11f3